spec/features/collaboration/user-collaboration_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
+
+feature 'User Collaboration', :type => :feature do
+
+  scenario 'add a collaborator' do
+    owner = create(:user, email: "test@email.com", password: "test-password")
+    collaborator = create(:user, email: "collaborator@email.com", password: "test-password")
+    wiki = create(:wiki, title: "Sports", body: "Sports content for all your sporting needs.", user: owner, private: true)
+
+    sign_in(owner.email, owner.password)
+    visit wiki_path(wiki)
+    click_link "Collaborate"
+    expect(page).to have_css("h1", text: "Wiki collaboration")
+    fill_in "Email", with: collaborator.email
+    click_button "Collaborate"
+    expect(current_path).to eq(wiki_path(wiki))
+    expect(page).to have_content(collaborator.email)
+  end
+
+  scenario 'delete a collaborator' do
+    owner = create(:user, email: "test@email.com", password: "test-password")
+    collaborator = create(:user, email: "collaborator@email.com", password: "test-password")
+    wiki = create(:wiki, title: "Sports", body: "Sports content for all your sporting needs.", user: owner, private: true)
+
+    sign_in(owner.email, owner.password)
+    visit wiki_path(wiki)
+    click_link "Collaborate"
+    expect(page).to have_css("h1", text: "Wiki collaboration")
+    fill_in "Email", with: collaborator.email
+    click_button "Collaborate"
+    expect(current_path).to eq(wiki_path(wiki))
+    expect(page).to have_content(collaborator.email)
+
+    click_link "Delete collaborator"
```

Comment:
Line 34 seems to spit out an object when I run the test, I'm not sure why?
```
User Collaboration
  add a collaborator
#<User id: 1, first_name: nil, last_name: nil, email: "test@email.com", encrypted_password: "$2a$04$kCH9N20rb/4P8/P9n1QlJupz567skmG4k28v7qPbnbM...", reset_password_token: nil, reset_password_sent_at: nil, remember_created_at: nil, sign_in_count: 1, current_sign_in_at: "2015-04-08 21:43:43", last_sign_in_at: "2015-04-08 21:43:43", current_sign_in_ip: "127.0.0.1", last_sign_in_ip: "127.0.0.1", confirmation_token: nil, confirmed_at: "2015-04-08 21:43:43", confirmation_sent_at: nil, unconfirmed_email: nil, created_at: "2015-04-08 21:43:43", updated_at: "2015-04-08 21:43:43", role: "standard">
#<User id: 1, first_name: nil, last_name: nil, email: "test@email.com", encrypted_password: "$2a$04$kCH9N20rb/4P8/P9n1QlJupz567skmG4k28v7qPbnbM...", reset_password_token: nil, reset_password_sent_at: nil, remember_created_at: nil, sign_in_count: 1, current_sign_in_at: "2015-04-08 21:43:43", last_sign_in_at: "2015-04-08 21:43:43", current_sign_in_ip: "127.0.0.1", last_sign_in_ip: "127.0.0.1", confirmation_token: nil, confirmed_at: "2015-04-08 21:43:43", confirmation_sent_at: nil, unconfirmed_email: nil, created_at: "2015-04-08 21:43:43", updated_at: "2015-04-08 21:43:43", role: "standard">
#<Collaboration id: 1, user_id: 2, wiki_id: 1, created_at: "2015-04-08 21:43:43", updated_at: "2015-04-08 21:43:43">
  delete a collaborator
```
Other than that runs as asked.